apps/web/playwright/booking/checkboxQuestione2e/checkbox-and-checkbox-question.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import { test } from "../../lib/fixtures";
+import { initialCommonSteps } from "../utils/bookingUtils";
+
+test.describe.configure({ mode: "parallel" });
```

Comment:
We already have `fullyParallel` set to `true` in the playwright config. Do we need this?

Reply:
Hi @keithwillcode we didn't know about this fullyParallel condition, we will remove this line in all PRs 😄